src/Api/Managers/NotificationManager.cs

```diff
@@ -0,0 +1,65 @@
+﻿using BLL.Managers;
+using BOL.Data;
+using BOL.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace BLL.Managers
+{
+    public class NotificationManager : BaseManager
+    {
+        public NotificationManager(CmsDbContext dbContext) : base(dbContext) { }
+
+        public virtual IEnumerable<Notification> CurrentUser(User userId)
+        {
+            return (from n in dbContext.Notifications
+                    where n.Id == userId.Id
+                    select n).ToList();
+        }
+
+        public bool UpdateNotification(int userId, int topicId, TopicFormModel model)
+        {
+            using (var transaction = dbContext.Database.BeginTransaction())
```

Comment:
Transactions are only necessary if you wan't to update several objects.
For adding you don't need one.